Cargo.toml

```diff
@@ -70,7 +70,7 @@ tracing-unstable = ["dep:tracing", "dep:log"]
 async-trait = "0.1.42"
 base64 = "0.13.0"
 bitflags = "1.1.0"
-bson = { git = "https://github.com/mongodb/bson-rust", branch = "main" }
+bson = { git = "https://github.com/mongodb/bson-rust", branch = "main", version = "2.11.0" }
```

Comment:
I mentioned adding a CI task to make sure that the driver compiles with these dependencies downloaded from crates.io rather than git. This is really just an issue for releases, though, and `cargo publish` will error if anything goes wrong there.